Tangem/Modules/Markets/UIComponents/MarketsEmptyAddTokenView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  MarketsEmptyAddTokenView.swift
+//  Tangem
+//
+//  Created by skibinalexander on 11.06.2024.
+//  Copyright © 2024 Tangem AG. All rights reserved.
+//
+
+import SwiftUI
+
+struct MarketsEmptyAddTokenView: View {
+    // MARK: - Properties
+
+    private(set) var didTapAction: (() -> Void)?
+
+    // MARK: - UI
+
+    var body: some View {
+        VStack(spacing: 12) {
+            headerView
+
+            buttonView
+        }
+        .padding(14)
+        .background(Colors.Background.primary)
+        .cornerRadiusContinuous(14)
```

Comment:
У нас есть это под белую подложку. Можешь там глянуть методы

```suggestion
        .roundedBackground(with: Colors.Background.primary, padding: 14)
```